Buildscripts/DevicetreeCompiler/source/generator.py

```
@@ -10,18 +10,25 @@ def write_include(file, include: IncludeC, verbose: bool):
     file.write('\n')
 
-def get_device_identifier_safe(device: Device):
-    if device.identifier == "/":
+def write_define(file, define: DefineC, verbose: bool):
+    if verbose:
+        print("Processing define:")
+        print(f"  {define.statement}")
+    file.write(define.statement)
+    file.write('\n')
+
+def get_device_node_name_safe(device: Device):
```

Ruff (0.14.14)

[error] `Device` may be undefined, or defined from star imports

(F405)